lib/find-python-script.py

```diff
@@ -0,0 +1,13 @@
+import sys, codecs;
+
+if (sys.stdout.encoding != "utf-8"):
+    if (sys.version_info[0] >= 3) and (sys.version_info[1] >= 7):
+        sys.stdout.reconfigure(encoding='utf-8')
+    else:
+        sys.stdout = codecs.getwriter("utf8")(sys.stdout)
+
+
+if (sys.version_info[0] >= 3) and (sys.version_info[1] >= 7):
+    print(sys.executable)
+else:
+    print(sys.executable.decode("cp1251"));
```

Comment:
Python files end with one carriage return so GitHub gets rid of that 🚫

Reply:
Do you know how to fix that?